events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+//  triviaMasterClientStandard.js
+//
+//  Created by Rebecca Stankus on 06/11/18
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This is the standard version of "Last Avatar Standing" Trivia 
+//  that draws from a third-party database of thousands of questions.
+
+/* global EventBridge Users AccountServices */
+
+(function() {
+
+    var SECRETS = Script.require(Script.resolvePath('triviaInfo.json')),
```

Comment:
coding standards: declare/define variables as close as possible to where they are first used (unless that doesn't make sense)

Reply:
gah, this would be a total pain for this file. since we're not rearchitecting everything right now, don't bother.

Reply:
I will adhere to this going forward.